doc/tutorials/rdt_configuration.rst

```diff
@@ -257,10 +257,10 @@ Configure RDT for VM Using VM Configuration
 
    .. code-block:: none
 
-      $ make hypervisor BOARD=apl-up2 FIRMWARE=uefi
+      $ make hypervisor BOARD=tgl-rvp FIRMWARE=uefi
```

Comment:
The `FIRMWARE` parameter is no longer valid. We need to remove it and update this with an appropriate pair of `SCENARIO` and `BOARD` parameters

Reply:
change to a board in the supported hardware document

Reply:
Updated.